src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -1128,16 +1307,23 @@ void Replayer<I>::regular_snapshot_complete(
     ceph_assert(r == 0);
     comp->release();
   } else {
+    dout(10) << "local group snap info: "
+          << "id: " << local_snap.id
+          << ", name: " << local_snap.name
+          << ", state: " << local_snap.state
+          << ", snaps.size: " << local_snap.snaps.size()
+          << dendl;
+    locker.unlock();
     on_finish->complete(0);
+    return;
```

Comment:
We don't have to log when `remote_snap.snaps.size() != local_image_snap_specs.size()`